### PR TITLE
Enable ec2-api ssl when want_all_ssl is set in mkcloud

### PR DESCRIPTION
**Pull request by sayalilunkad**

### CODE REVIEW

**Comment by bmwiedemann** [comment]
does that work with SOC5/6 ?

**Comment by sayalilunkad** [comment]
@bmwiedemann nope, https://github.com/crowbar/crowbar-openstack/pull/773 is still to be merged

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -2167,6 +2167,7 @@ function enable_ssl_generic
         nova)
             $p "$a['ssl']['enabled']" true
             $p "$a['novnc']['ssl']['enabled']" true
+            $p "$a['ec2-api']['ssl']['enabled']" true
```
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: needs a `iscloudver 7plus &&`

**Comment by sayalilunkad** [comment]
@bmwiedemann can we merge this? We wanted to test the other PR with this setting.

File: scripts/qa_crowbarsetup.sh [hunk 2]
```diff
@@ -2167,6 +2167,9 @@ function enable_ssl_generic
         nova)
             $p "$a['ssl']['enabled']" true
             $p "$a['novnc']['ssl']['enabled']" true
+            if iscloudver 7plus ; then
+              $p "$a['ec2-api']['ssl']['enabled']" true
```
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: needs to be indented with 4 spaces to pass the 'bashate' style-checker

**Comment by bmwiedemann** [comment]
@sayalilunkad nay, we need to re-test this one with
```
want_all_ssl=1
UPDATEREPOS=http://clouddata.cloud.suse.de/mkcloud/crowbar-openstack:773:e87b1191ed641ddfa3f7271b51c05a630757d9c1:master/
```

**Comment by bmwiedemann** [comment]
Test is running with ssl and the crowbar change included.

**Pull request merged**